ScrabbleUnitTest/ScrabbleUnitTest.csproj

```diff
@@ -50,10 +50,10 @@
     </Otherwise>
   </Choose>
   <ItemGroup>
-    <Compile Include="FirstThingsFirstUnitTests.cs" />
     <Compile Include="Game Objects\GameBoardSquareTests.cs" />
     <Compile Include="Game Objects\LetterTileRackTests.cs" />
     <Compile Include="Game Objects\LetterTileTests.cs" />
+    <Compile Include="PlayerClass\PlayerTests.cs" />
```

Comment:
Look for your merge conflicts in this file. Open up this file in a text editor. Have you tried merging your branch into the recent sh4ne/master?